telegram/constants.py

```diff
@@ -1754,3 +2587,50 @@ class ForumTopicLimit(IntEnum):
     * :paramref:`~telegram.Bot.edit_general_forum_topic.name` parameter of
       :meth:`telegram.Bot.edit_general_forum_topic`
     """
+
+
+class Date(Enum):
+    """This enum contains the date literal for :class:`telegram.InaccessibleMessage`
+
+    .. versionadded:: NEXT.VERSION
+    """
+
+    __slots__ = ()
+
+    ZERO_DATE = datetime.datetime(1970, 1, 1, tzinfo=UTC)
+    """:obj:`datetime.datetime`, value of unix 0."""
+
+
+class ReactionType(StringEnum):
+    """This enum contains the available types of :class:`telegram.Update`. The enum
+    members of this enumeration are instances of :class:`str` and can be treated as such.
+
+    .. versionadded:: NEXT.VERSION
+    """
+
+    __slots__ = ()
+
+    EMOJI = "emoji"
+    """:obj:`str`: A :class:`telegram.ReactionType` with a normal emoji."""
+    CUSTOM_EMOJI = "custom_emoji"
+    """:obj:`str`: A :class:`telegram.ReactionType` with a custom emoji."""
+
+
+class ReactionEmojis(StringEnum):
+    """This enum contains the available emojis of :class:`telegram.ReactionTypeEmoji`. The enum
+    members of this enumeration are instances of :class:`str` and can be treated as such.
+
+    .. versionadded:: NEXT.VERSION
+
+    The descriptions were taken from emojipedia.org
+    """
+
+    # TODO if we want to have these descriptions, we should ask for permission from emojipedia
```

Comment:
Interesting! Maybe the unicode consortium also provides a description? I would assume that those are intended for public use … In any case I don't think that we'll run into trouble …

Reply:
they do not only the official names

Reply:
The [CLDR names](https://cldr.unicode.org/index) seem to be the names that are listed on emojipedia as "unicode name". The data can be downloaded [here](https://cldr.unicode.org/index/downloads), which points to https://github.com/unicode-org/cldr, which has a [do-whatever-you-want license](https://github.com/unicode-org/cldr/blob/main/LICENSE).

The cldr names are also listed on https://www.unicode.org/emoji/charts/full-emoji-list.html